packages/manager/modules/pci/src/projects/project/users/users.service.js

```diff
@@ -85,6 +85,17 @@ export default class PciProjectsProjectUsersService {
     });
   }
 
+  getS3StorageRegions(projectId, regionCapacity) {
+    return this.getRegions(projectId).then((regions) => {
+      return regions.filter(({ services }) =>
+        services.find(
+          ({ name }) =>
+            name === regionCapacity[0] || name === regionCapacity[1],
+        ),
+      );
+    });
+  }
```

Comment:
**maintainability:** This is repeated in `packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js`. Can be maintained in a common service